src/app/domain/backup/services/getMedias.ts

```diff
@@ -27,6 +27,36 @@ const getPhotoIdentifiersPage = async (
   return photoIdentifiersPage
 }
 
+export const getMimeType = (media: Media): string => {
+  if (media.mimeType) {
+    return media.mimeType
+  }
+
+  const extension = media.name
+    .substring(media.name.lastIndexOf('.') + 1)
+    .toLowerCase()
+
+  if (media.type === 'image') {
+    switch (extension) {
+      case 'jpg':
+        return 'image/jpeg'
+      default:
+        return `image/${extension}`
+    }
+  }
+
+  if (media.type === 'video') {
+    switch (extension) {
+      case 'mp4':
+        return 'video/mpeg'
+      default:
+        return `video/${extension}`
```

Comment:
Are we sure to want that? Having video/ogx will be weird (instead of video/ogg). Do we really need to pass the extension?

Reply:
How do Drive handle that?

Reply:
I add 'mime/lite' to have the best mime guessing. It is 2.5KB. It will be possible to remove it one day if cameraroll return mime type for iOS. 

Package is also used by Drive for some purposes.